simplq/src/components/Layout/layout.module.scss

```diff
@@ -8,8 +8,6 @@
 
   .content {
     flex: 1 1 auto;
-    display: flex;
-    flex-direction: column;
   }
```

Comment:
I'm curious, why was this only a problem on Safari? Do they not use the latest CSS guidelines or something?

Reply:
I don't know. I don't know why we added the two lines either.